lib/handler/status/events.c

```diff
@@ -66,29 +66,29 @@ static h2o_iovec_t errors_status_final(void *priv, h2o_globalconf_t *gconf, h2o_
 #define BUFSIZE (2*1024)
     ret.base = h2o_mem_alloc_pool(&req->pool, BUFSIZE);
     ret.len = snprintf(ret.base, BUFSIZE, ",\n"
-                                          " \"http1-errors-400\": %" PRIu64 ",\n"
-                                          " \"http1-errors-403\": %" PRIu64 ",\n"
-                                          " \"http1-errors-404\": %" PRIu64 ",\n"
-                                          " \"http1-errors-405\": %" PRIu64 ",\n"
-                                          " \"http1-errors-416\": %" PRIu64 ",\n"
-                                          " \"http1-errors-417\": %" PRIu64 ",\n"
-                                          " \"http1-errors-500\": %" PRIu64 ",\n"
-                                          " \"http1-errors-502\": %" PRIu64 ",\n"
-                                          " \"http1-errors-503\": %" PRIu64 ",\n"
-                                          " \"http2-errors-protocol\": %" PRIu64 ", \n"
-                                          " \"http2-errors-internal\": %" PRIu64 ", \n"
-                                          " \"http2-errors-flow_control\": %" PRIu64 ", \n"
-                                          " \"http2-errors-settings_timeout\": %" PRIu64 ", \n"
-                                          " \"http2-errors-stream_closed\": %" PRIu64 ", \n"
-                                          " \"http2-errors-frame_size\": %" PRIu64 ", \n"
-                                          " \"http2-errors-refused_stream\": %" PRIu64 ", \n"
-                                          " \"http2-errors-cancel\": %" PRIu64 ", \n"
-                                          " \"http2-errors-compression\": %" PRIu64 ", \n"
-                                          " \"http2-errors-connect\": %" PRIu64 ", \n"
-                                          " \"http2-errors-enhance_your_calm\": %" PRIu64 ", \n"
-                                          " \"http2-errors-inadequate_security\": %" PRIu64 ", \n"
-                                          " \"http2-read-closed\": %" PRIu64 ", \n"
-                                          " \"http2-write-closed\": %" PRIu64 "\n",
+                                          " \"http1-errors.400\": %" PRIu64 ",\n"
+                                          " \"http1-errors.403\": %" PRIu64 ",\n"
+                                          " \"http1-errors.404\": %" PRIu64 ",\n"
+                                          " \"http1-errors.405\": %" PRIu64 ",\n"
+                                          " \"http1-errors.416\": %" PRIu64 ",\n"
+                                          " \"http1-errors.417\": %" PRIu64 ",\n"
+                                          " \"http1-errors.500\": %" PRIu64 ",\n"
+                                          " \"http1-errors.502\": %" PRIu64 ",\n"
+                                          " \"http1-errors.503\": %" PRIu64 ",\n"
```

Comment:
Please rename the errors to `status-errors`, as discussed in https://github.com/h2o/h2o/pull/893#commitcomment-17607549.

I believe that is the only change required to getting this merged.  Thank you!